### ISSUE TITLE
Feature Request: Output if deployed existing version

### ISSUE THREAD

**Issue by williamscs** [einaregilsson/beanstalk-deploy#26]
I think it'd be useful to know if we created a new application version or if this action used a version that already existed.

In my case, I have a deployment pipeline that also increments the version. It'd be useful to use the same pipeline to redeploy old versions, but not increment the version if this action didn't deploy any new artifacts.

**Comment by einaregilsson** [comment]
How do you see that working? Let the action have some kind of output that tells whether a version was created?

**Comment by williamscs** [comment]
Yeah, I think so. The check already exists to see if that application version already exists and the app chooses to use the existing version. https://github.com/einaregilsson/beanstalk-deploy/blob/02687fd5e0f908b502fea1d44fb1b2ab04af2d62/beanstalk-deploy.js#L318-L341

At that point, an output could be populated with a boolean based on which direction it took. Or just output the `versionAlreadyExists` variable?

**Comment by einaregilsson** [comment]
Should be easy enough to add it as an output I guess. Not working on this action right now, but will try to get a new version going in the next couple of weeks.
